libkbfs/folder_branch_ops.go

```diff
@@ -1317,6 +1317,10 @@ func (fbo *folderBranchOps) SetInitialHeadFromServer(
 		return errors.Errorf("MD with revision=%d not initialized",
 			md.Revision())
 	}
+	fbo.config.
+		BlockOps().
+		Prefetcher().
+		PrefetchDirBlock(md.data.Dir.BlockPointer, md, dirEntryPrefetchPriority)
```

Comment:
I'll add a TODO to do that in the next PR. There's some more plumbing that needs to go in to support that, and I'm already setting myself up for conflict hell as is with some of the refactors I've done.